web_scraping/gog_scraper/scrape_gog_game.py

```diff
@@ -2,9 +2,12 @@
 
 from datetime import datetime
 from urllib.request import urlopen
+import logging
```

Comment:
Great idea mehrdad

Reply:
Next time will know to do this from the start :')